cli/js/web/url.ts

```diff
@@ -298,7 +299,9 @@ export class URLImpl implements URL {
     value = String(value);
     try {
       parts.get(this)!.hostname = encodeHostname(value);
-    } catch {}
+    } catch {
+      // pass
+    }
```

Comment:
There's no point to this style. It's more lines and necessitates a comment that says "pass".